mon-pix/translations/fr.json

```
@@ -3,21 +3,21 @@
         "bad-request-error": "Les données que vous avez soumises ne sont pas au bon format.",
         "internal-server-error": "Une erreur interne est survenue, nos équipes sont en train de résoudre le problème. Veuillez réessayer ultérieurement.",
         "join-error": {
-            "r11": "Vous possédez déjà un compte Pix avec l’adresse e-mail '<br>'{ value }.'<br>'Pour continuer, connectez-vous à ce compte ou demandez de l’aide à un enseignant.'<br>'(Code R11)",
-            "r12": "Vous possédez déjà un compte Pix utilisé avec l’identifiant '<br>'{ value }.'<br>'Pour continuer, connectez-vous à ce compte ou demandez de l’aide à un enseignant.'<br>'(Code R12)",
+            "r11": "Vous possédez déjà un compte Pix avec l’adresse e-mail '<b>'{ value }.'</b>'Pour continuer, connectez-vous à ce compte ou demandez de l’aide à un enseignant.'<br>'(Code R11)",
```

Pourquoi passer de `br` à `b` ? On ne va pas avoir le même résultat (on perd le retour à la ligne). Peut être à voir avec les PO pour ce cas là

---

`br` fait un retour à la ligne, or je trouve bizarre un retour à la lign juste pour mettre `{ value }.`

Autre suggestion, mettre un br **après** la phrase :
```suggestion
            "r11": "Vous possédez déjà un compte Pix avec l’adresse e-mail '<b>'{ value }.'</b><br>'Pour continuer, connectez-vous à ce compte ou demandez de l’aide à un enseignant.'<br>'(Code R11)",
```

---

Ca fait un changement d'affichage coté front, je pinguerait peut être emmy sur la question (ou accèsà

---

La team-prescription garde en tête cette discussion mais ne pense pas que cela mettra suffisamment en évidence la partie du texte en le mettant simplement en gras.
Je supprime cette modifiaction du coup.